include/wpe/view-backend.h

```diff
@@ -166,9 +166,7 @@ struct wpe_view_backend_client {
     void (*activity_state_changed)(void*, uint32_t);
     void* (*get_accessible)(void*);
     void (*set_device_scale_factor)(void*, float);
-
-    /*< private >*/
-    void (*_wpe_reserved0)(void);
+    void (*target_refresh_rate_changed)(void*, uint32_t);
```

Comment:
Next time we need to add something we'll need an API/ABI bump. We have been getting a fairly good mileage out of API 1.0, so no regrets! 😄